### PR TITLE
Add new CEF parser plugin

### PR DESCRIPTION
**Pull request by jachris**
We can now `parse cef` when using the new pipelines :tada:

### CODE REVIEW

File: changelog/next/features/3110--cef-parser.md [hunk 1]
```diff
@@ -0,0 +1 @@
+The `cef` parser allows using the CEF format with the new pipelines.
```
Comment (dominiklohmann) [inline changelog/next/features/3110--cef-parser.md:1]:
```suggestion
The `cef` parser allows for using the CEF format with the new pipelines.
```

File: libvast/builtins/formats/cef.cpp [hunk 2]
```diff
@@ -6,6 +6,8 @@
 // SPDX-FileCopyrightText: (c) 2022 The VAST Contributors
 // SPDX-License-Identifier: BSD-3-Clause
 
+#include "vast/to_lines.hpp"
```
Comment (dominiklohmann) [inline libvast/builtins/formats/cef.cpp:9]:
```suggestion
#include <vast/to_lines.hpp>
```

Mostly a nit; can we maybe enforce this with a clang-format rule for the builtins?

File: changelog/next/features/3110--cef-parser.md (hunk 1 above)
Comment (dominiklohmann) [inline changelog/next/features/3110--cef-parser.md:1]: Please add an integration test based on `vast exec` for this.

**Review by dominiklohmann — APPROVED** [review]
🚀

File: libvast/builtins/formats/cef.cpp (hunk 2 above)
Comment (jachris) [reply libvast/builtins/formats/cef.cpp]: We can add a `.clang-format` file to `libvast/builtins` with `BasedOnStyle: InheritParentConfig`, but I think `clang-format` can only changes the order of includes.

Comment (dominiklohmann) [reply libvast/builtins/formats/cef.cpp]: There's an in-review patch here that would somewhat fix this, but nothing we can use now: https://reviews.llvm.org/D145843

**Pull request merged**